bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,229 @@
+# airq Binding
+
+The airq Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+<img src="https://uploads-ssl.webflow.com/5bd9feee2fb42232fe1d0196/5e4a8dc0e322ca33891b51e4_air-Q%20frontal-p-800.png" alt="air-Q image" width="400px" height="324px" />
+
+## Supported Things
+
+One only Thing is supported: the air-Q device.
+This Binding was tested with an air-Q Pro device with 14 sensors. It should also work with an air-Q device with 11 sensors, but it was not tested yet.
+
+## Discovery
+
+Auto-discovery is not possible in this version. Since the binding has to be configured at least with the password of the device, auto-discovery would be of limited value anyway.
```

Comment:
It is fine to not implement discovery, however is it possible to add discovery and add the device without the password? It should then just go OFFLINE and have a status that the password is wrong/missing. The user then just edits the thing to have a password and it can then go ONLINE.

Reply:
The implementation of the discovery on the air-Q is something odd, it does not answer to multi- or broadcasts. At this moment, I would like to publish the binding without the discovery and implement it later, once I get the appropriate information from the air-Q manufacturers. So at the moment it does not work without giving the IP address to the thing.